controllers/topics.controllers.js

```diff
@@ -0,0 +1,11 @@
+const { fetchTopics } = require("../models/topics.models.js");
+
+exports.getTopics = (req, res, next) => {
+  fetchTopics()
+    .then((topics) => {
+      res.status(200).send(topics);
```

Comment:
This should be sent on an object with a key of topics, and a value of the array full of topics. Please update your tests to reflect this too!